client/post-editor/editor-diff-viewer/index.jsx

```diff
@@ -8,41 +8,45 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import classNames from 'classnames';
 import { connect } from 'react-redux';
-import { get, map } from 'lodash';
+import { map } from 'lodash';
 
 /**
  * Internal dependencies
  */
-import { getPostRevision, getPostRevisionChanges } from 'state/selectors';
+import { getPostRevisionChanges } from 'state/selectors';
 
-const EditorDiffViewer = ( { contentChanges, revision } ) => (
+const renderHighlightedChanges = changes =>
```

Comment:
It may be a good idea to create a new component so that we could do the following inline of the render body, what do you think? I don't feel super strongly on this though, happy with it as is :)
```jsx
map( changes, change =>
  <HighlightedChange change={ change } key={ index } />
);
```

Reply:
I mainly wanted to extract the looping part, that's why I moved the whole thing a level higher. 

I've pushed an update now that transforms the "random function" into its own component (with suggested improvement for React 16). What do you think about that one?